src/pages/vault/tutorials/[collectionSlug].tsx

```diff
@@ -7,9 +7,8 @@ import {
   getCollectionPaths,
   CollectionPageProps,
 } from 'views/collection-view/server'
-import BaseLayout from 'layouts/base-new'
 
-export function VaultCollectionPage(
+export default function VaultCollectionPage(
```

Comment:
@kendallstrautman what I was suggesting here is that this (and the one for Waypoint):

```tsx
export default function VaultCollectionPage(
  props: CollectionPageProps
): React.ReactElement {
  return <CollectionView {...props} />
}
```

get replaced with this:

```tsx
export default CollectionView
```

Which is what a few product landing pages do with `ProductLandingView`:

- [Vault](https://github.com/hashicorp/dev-portal/blob/main/src/pages/vault/index.tsx#L16)
- [Waypoint](https://github.com/hashicorp/dev-portal/blob/main/src/pages/waypoint/index.tsx#L16)